frontend/public/components/events.jsx

```diff
@@ -227,7 +227,10 @@ export class EventsList extends React.Component {
         <Helmet>
           <title>Events</title>
         </Helmet>
-        <PageHeading detail={true} title="Events">
+        <PageHeading
+          detail={true}
+          title={this.props.pageTitle !== undefined ? this.props.pageTitle : 'Events'}
```

Comment:
This feels a bit backwards but it works.
I was thinking that perhaps `EventStreamPage` should set the title, and by default this component doesn't use a title. Therefore in `MonitoringEvents` you won't need to specify an empty `pageTitle` prop.
Prop name `title` would be more concise.

Reply:
Modified `EventStreamPage` to set the title.